asuka-core/src/stores/sqlite.rs

```diff
@@ -13,7 +14,6 @@ use zerocopy::IntoBytes;
 pub struct Account {
     pub id: i64,
     pub name: String,
-    pub source_id: String,
```

Comment:
this is intended to be a stable identifier from the account source since in most cases you can change your name